plugins/inputs/nfsstat/nfsstat.go

```diff
@@ -0,0 +1,289 @@
+package nfsstat
+
+import (
+	"strconv"
+	"strings"
+
+	"errors"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io/ioutil"
+)
+
+var (
+	ErrorInvalidNFSVersion   error = errors.New("invalid nfs version")
+	ErrorInvalidNFSOpIndex   error = errors.New("invalid nfs operation index")
+	ErrorInvalidCounterValue error = errors.New("invalid counter value")
+)
+
+const (
+	prefix_nfs3 string = "proc3 22 "
+	prefix_nfs4 string = "proc4 60 "
+)
+
+type NFSStat struct {
+	Log telegraf.Logger
+}
+
+type NFSStatCounter struct {
+	nfsvers string
+	op      string
+	val     int64
+}
+
+func (n *NFSStat) SampleConfig() string {
+	return "[[inputs.nfsstat]]"
```

Comment:
SampleConfig() shouldn't include the name of the plugin
```suggestion
	return ""
```